macros/union/github_union_connections.sql

```diff
@@ -0,0 +1,93 @@
+{% macro github_union_connections(connection_dictionary, single_source_name, single_table_name, default_identifier=single_table_name) %}
+
+{{ adapter.dispatch('github_union_connections', 'github') (connection_dictionary, single_source_name, single_table_name, default_identifier) }}
+
+{%- endmacro %}
+
+{% macro default__github_union_connections(connection_dictionary, single_source_name, single_table_name, default_identifier=single_table_name) %}
```

Comment:
I cleaned up this macro a bit to help our future selves when doing maintenance. I mainly made vars for repeated items to help with readability and understanding.